src/features/game/state/GameStateManager.ts

```diff
@@ -454,6 +458,55 @@ class GameStateManager {
     return this.checkpointTask.getAllVisibleTaskData();
   }
 
+  ///////////////////////////////
+  //          Quiz             //
+  ///////////////////////////////
+
+  /**
+   * Checks whether a quiz has been obtained full marks.
+   *
+   * @param key quiz id
+   * @returns {boolean}
+   */
+  public isQuizComplete(quizId: string): boolean {
+    return this.quizScores.get(quizId) === GameGlobalAPI.getInstance().getQuizLength(quizId);
+  }
+
+  /**
+   * Checks whether a specific quiz has been played.
+   *
+   * @param key quiz id
+   * @returns {boolean}
+   */
+  public isQuizAttempted(quizId: string): boolean {
```

Comment:
Ditto.

Reply:
The method is used in GameGlobalAPI and GameActionConditionChecker

Reply:
Resolving the conversation for now as changing access is needed for `GameGlobalAPI`. (GameGlobalAPI is a separate issue that is out of scope for this PR 😓)